app/lib/reducers/ui.js

```diff
@@ -24,7 +24,15 @@ const initial = Immutable({
   fontFamily: 'Menlo, "DejaVu Sans Mono", "Lucida Console", monospace',
   fontSizeOverride: null,
   css: '',
-  termCSS: '',
+  termCSS: '' +
```

Comment:
You can certainly use template strings btw, it's natively supported by the node used by electron.